lib/vmfloaty.rb

```diff
@@ -100,10 +101,18 @@ def run # rubocop:disable Metrics/AbcSize
           running_vms = service.list_active(verbose)
           host = URI.parse(service.url).host
           if running_vms.empty?
-            puts "You have no running VMs on #{host}"
+            if options.json
+              puts {}.to_json
+            else
+              puts "You have no running VMs on #{host}"
```

Comment:
This change just got merged so you might need to rebase off master...but it'd be good to move these user facing `puts` messages to the `FloatyLogger` class instead of using puts here. I think maybe just `puts`ing JSON above might be ok? :shrug: